pylcurve/lcurve.py

```diff
@@ -124,11 +150,13 @@ def write(self, fname=None):
         fptr.close()
         return fname
 
-    def run(self, data):
+
+    def run(self, data, scale_factor=None):
         """
-        Run lroche, return output and name of model
+        Run lroche, return output and name of model.
         """
         data = str(data)
+        scale_factor = str(scale_factor)
```

Comment:
How does this work for scale_factor=None?

Reply:
I don't think the string conversion is needed

Reply:
Fixed (I think), the string conversion was accidentally left behind after i put format in there